omero/developers/Web/Deployment.txt

```diff
@@ -58,142 +58,7 @@ Debugging on and start the server up:
 Using FastCGI (Unix/Linux)
 --------------------------
 
-Once you have installed a FastCGI capable web server, configuration of
```

Comment:
I moved that section to the bottom of the doc, and it doesn't mention the `--system` flag at all.